docs/static/releasenotes.asciidoc

```diff
@@ -1,73 +1,56 @@
 [[releasenotes]]
 == Release Notes
 
-This section summarizes the changes in each release.
-
-* <<RC1, Logstash 5.0.0-rc1>>
-* <<beta1, Logstash 5.0.0-beta1>>
-* <<alpha5,Logstash 5.0.0-alpha5>>
-* <<alpha4,Logstash 5.0.0-alpha4>>
-* <<alpha3,Logstash 5.0.0-alpha3>>
-* <<alpha2,Logstash 5.0.0-alpha2>>
-* <<alpha1,Logstash 5.0.0-alpha1>>
-
-[[RC1]]
-=== Logstash 5.0.0-rc1 Release Notes
-
-* Windows: Fixed signal HUP warnings on startup ({lsissue}5239[Issue 5239]).
-* Windows: Enabled logging on Windows by fixing log4j config location ({lsissue}5971[Issue 5971]).
-* Configuration errors are now logged to file, not printed to stderr ({lsissue}5975[Issue 5975]).
-* Correctly calculate pipeline `duration_in_millis` metric ({lsissue}5952[Issue 5952]).
+This section summarizes the changes in {version} release.
```

Comment:
I wouldn't use a variable here because you'll want different versions when you add release notes for subsequent point releases.